src/test/scala/com/picnicml/doddlemodel/metrics/RegressionMetricsTest.scala

```diff
@@ -11,4 +11,18 @@ class RegressionMetricsTest extends FlatSpec with Matchers {
 
     rmse(y, yPred) shouldEqual 3.076686529368892
   }
+
+  "Regression metrics" should "calculate a correct mae value" in {
+    val y = DenseVector(1.0, 4.1, 2.2, 5.1, 9.6)
+    val yPred = DenseVector(1.2, 1.9, 2.8, 4.1, 10.6)
+
+    assert(mae(y, yPred) === 1.0 +- 0.0000001)
+  }
+
+  "Regression metrics" should "calculate the correct explained variance score" in {
```

Comment:
You can use the `it` word to avoid retyping `"Regression metrics"` over and over (e.g. take a look [here](https://github.com/evanhaldane/doddle-model/blob/ba2fee2e083cbdd1e611d935ddbcd31683a27eea/src/test/scala/com/picnicml/doddlemodel/linear/LinearRegressionTest.scala#L22)).